Uplift/Uplift.csproj

```diff
@@ -8,6 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Marten" Version="7.28.0" />
```

Comment:
Why do we need this package?

Reply:
You are Right. in this course there are some lessons that was for version 2 of 3 in .NET Core. we do not need it.